projects/storefrontlib/cms-components/product/carousel/product-carousel/product-carousel.module.ts

```diff
@@ -21,6 +23,7 @@ import { ProductCarouselComponent } from './product-carousel.component';
     RouterModule,
     UrlModule,
     I18nModule,
+    PageComponentModule,
```

Comment:
What is this used for?

Reply:
This exports `InnerComponentsHostDirective`, which is used in the declaration of `ProductCarouselItemDirective`.

Reply:
`InnerComponentsHostDirective`, thank!